src/vue/backend/src/App.vue

```diff
@@ -1,12 +1,81 @@
 <template>
-  <idx-block className="app">
-    <idx-block className="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
+    <idx-block className="app {'nav-is-expanded': expanded}">
+        <Header/>
+        <Navigation>
+            <idx-block className="nav">
+                <router-link to="/settings">Settings</router-link>
+            </idx-block>
+        </Navigation>
+        <Page>
+            <router-view/>
+        </Page>
     </idx-block>
-    <router-view/>
-  </idx-block>
 </template>
 
+<script>
+import Header from '@/components/Header.vue'
+import Navigation from '@/components/Navigation.vue'
+import Page from '@/components/Page.vue'
+
+export default {
+    components: {
+        Header,
+        Navigation,
+        Page
+    },
+    data () {
+        return {
+            expanded: true
+        }
+    },
+    methods: {
+    },
+    computed: {
+    },
+    created () {
+    }
+}
+</script>
+
 <style lang="scss">
+    @import '~@idxbrokerllc/idxstrap/dist/styles/globalVariables.scss';
```

Comment:
You should not need `globalVariables.scss` because it's loaded in globally in the vue config file: https://github.com/idxbroker/wordpress-plugin/blob/release/3.0.0/src/vue/backend/vue.config.js#L33